internal/globaldefault/default_lin_mac.go

```diff
@@ -0,0 +1,8 @@
+// +build !windows
+
+package globaldefault
+
+func isOnPATH(binDir string) bool {
+	// todo: https://www.pivotaltracker.com/story/show/177673816
```

Comment:
It took me a while until I understood what you want to achieve here: You actually want to have a way to figure out whether we have already called `subshell.WriteUserEnv(..., envType)` with that PATH before, correct?

Reply:
Correct.